category.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace PHPMD\Category;
+
+class Category
+{
+    public static $checks;
+}
+
+Category::$checks = array(
```

Comment:
I'd also probably add methods on `Category` for pulling out the mapped data, instead of exposing the underlying structure directly.